src/lib/process/map.ts

```diff
@@ -4,23 +4,27 @@ import type { ChainRestrictions } from '../types/restrictions.js'
 
 // Changes more human-readable format of grouping the same type of restriction -> an array of single restrictions for submitting to node
 // { RestrictionName: [RestrictionValue] } -> [ { RestrictionName: RestrictionValue } ]
-export const mapRestrictions = (rawRestrictions: string): ChainRestrictions => {
-  const restrictionsObj: Object = userRestrictionValidation.parse(JSON.parse(rawRestrictions))
-
-  const restrictionPairs = Object.entries(restrictionsObj)
-  if (restrictionPairs.length === 0) {
+export const mapRestrictions = (rawRestrictions: string): any => {
```

Comment:
Need to remove `any` usage. This is quite an important data transform

Reply:
agreed

Reply:
I believe this is no longer relevant due to logic update, marking as resolved